src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/services/PlanService.kt

```diff
@@ -23,7 +23,7 @@ import uk.gov.justice.digital.hmpps.sentenceplan.entity.request.SignType
 import uk.gov.justice.digital.hmpps.sentenceplan.entity.response.SoftDeletePlanVersionsResponse
 import uk.gov.justice.digital.hmpps.sentenceplan.exceptions.ConflictException
 import java.time.LocalDateTime
-import java.util.UUID
+import java.util.*
```

Comment:
Can you put this back to `non-star` import?

Reply:
Seeing as though Intellij and Jetbrains created the Kotlin language, the linter intellij providers is probably correct. Wildcard Imports are actually okay. The concept that they are bad should die with Java. Ive changed anyway